lib/services/github-types.d.ts

```diff
@@ -22,24 +22,22 @@ import { ServiceAPIHandle, ServiceEmitContext, ServiceEvent, ServiceListenerMeth
 	ServiceType } from './service-types';
 
 /** The Github service can be used as either an Integration or a User. */
-type GithubLoginType = 'integration' | 'user';
+type GithubLoginType = 'app' | 'pat';
 
 /** Allows a service to login to Github as an Integration, using the ID. */
-export interface GithubIntegration {
-	/** Integration ID of the Github Integration to run as. */
-	integrationId: number;
+export interface GithubApp {
+	/** App ID of the Github Integration to run as. */
+	appId: number;
 	/** The Github PEM as an encoded Base64 value. */
 	pem: string;
 	/** Type of service ('listener' or 'emitter'). */
 	type: GithubLoginType;
 }
 
 /** Allows a service to login to Github as a user, using username and Personal Access Token. */
-export interface GithubUser {
+export interface GithubPAT {
 	/** The PAT for the user to login as. */
 	pat: string;
-	/** The username of the user to login as. */
-	user: string;
 	/** Type of service ('listener' or 'emitter'). */
 	type: GithubLoginType;
```

Comment:
This comment (and the equivalent one for `GithubApp`) is now wrong I think - it looks like this is now 'app' | 'pat'

In addition, it's not even that: for each type it's _always_ one of those: `type: 'pat'` here and `type: 'app'` for `GithubApp`.